provision/infinispan/ispn-helm/templates/infinispan.yaml

```diff
@@ -0,0 +1,92 @@
+{{- $namespace := .Values.namespace | required ".Values.namespace is required." -}}
+apiVersion: infinispan.org/v1
+kind: Infinispan
+metadata:
+  name: infinispan
+  namespace: {{ $namespace }}
+  annotations:
+    infinispan.org/monitoring: 'true'
+spec:
```

Comment:
We can reduce the footprint of the clusters by disabling the ConfigListener. As we're defining all caches statically, and users can't create new caches at runtime, there's no benefit of enabling this.

```yaml
spec:
  configListener:
    enabled: false
```

Reply:
Added!